docs/version0.33.md

```diff
@@ -50,6 +51,10 @@ This was already required for OpenJDK 17 and is now also required from 11.0.16 t
 
 Linux builds for all platforms now use gcc 10.3 instead of gcc 7.5. See the list of [build environments](openj9_support.md#build-environments).
 
+### The maximum number of compilation threads that are used by the JIT compiler increased
```

Comment:
The title is odd because it switches from full English to brief English. I suggest something more like "The maximum number of JIT compilation threads is increased"